### DISCUSSION TITLE
How do I build tremor for testing a PR?

### DISCUSSION THREAD

**Discussion by krisalyssa** [tremorlabs/tremor-npm#296] [Q&A] [answered]
I know how to reference tremor from my app as a local file dependency (at least I think so -- following https://docs.npmjs.com/cli/v6/configuring-npm/package-json#local-paths). I've built at least some of the output files by commenting out `emitDeclarationOnly` in `tsconfig.json` and running `npx tsc`, but at least one file is still missing:
```
error - ./src/pages/_app.tsx:2:0
Module not found: Can't resolve '@tremor/react/dist/esm/tremor.css'
```
And sure enough, there's no `esm` directory in `dist`.

I'm happy to open a PR with changes to `README.md` once I know what should be in there.

**Comment by mitrotasios** [comment, marked as answer]
hey @CraigCottingham, I am not sure I am following. If you like testing a PR within the tremor repo I recommend running `npm run rollup` (in that case `dist` should include a `esm` directory) to build and `npm run storybook` to test the components on storybook 🙂

**Comment by krisalyssa** [comment]
I think `npm run rollup` is what I was looking for. I have an app that has `tremor` as a dependency, and I want to be able to test my changes to `tremor` in that app before I open a PR here. I'll let you know once I've had a chance to confirm.

**Reply by mitrotasios** [reply]
Awesome, thanks!
